src/client/sandbox/event/simulator.ts

```diff
@@ -468,16 +472,54 @@ export default class EventSimulator {
         return null;
     }
 
+    _getPointerEventTypeInfo (type: string) : { eventType: string, pointerType: string } {
+        const mouseToPointerEventTypeMap = {
```

Comment:
Move `mouseToPointerEventTypeMap` and `touchToPointerEventTypeMap` constans on the module level.